eclipse-collections/src/main/java/org/eclipse/collections/impl/collection/mutable/AbstractMultiReaderMutableCollection.java

```diff
@@ -77,10 +77,11 @@
  * AbstractMultiReaderMutableCollection is a common abstraction that provides thread-safe collection behaviors.
  * Subclasses of this class must provide implementations of getDelegate() and getLock().
  */
+@SuppressWarnings("unused")
 public abstract class AbstractMultiReaderMutableCollection<T> implements MutableCollection<T>
 {
-    protected transient ReadWriteLock lock;
-    protected transient ReadWriteLockWrapper lockWrapper;
+    protected ReadWriteLock lock;
```

Comment:
Why is it ok to remove transient here?

Reply:
`transient` only affects `Serializable` classes. `AbstractMultiReaderMutableCollection` and its subclasses are not `Serializable`. Its subclasses are `Externalizable`

In theory, someone could write a new serializable subclass of AbstractMultiReaderMutableCollection. If you would prefer, I could bring back the keyword and suppress the warning.

Reply:
Yes, bring back the keyword and suppress the warning.